### DISCUSSION TITLE
Training w/ WAV Files Converted From MP3?

### DISCUSSION THREAD

**Discussion by automationist** [coqui-ai/TTS#1268] [General]
I'm wondering which one of these options would result in a better model:
1. Adding more data (MP3)
2. Not adding the MP3 data, and only training with the data that has never been compressed. 

What do you guys think?

**Comment by thorstenMueller** [comment]
Not being an audio expert, but i would use the original wavefiles (_if you have access to them_). Compressing to mp3 might remove audio details that will not come back by converting back to wave. If that's your question.

**Comment by domcross** [comment]
When the MP3s have high quality (bitrate of 192KBits/sec or higher) it should be good enough to give it a try. As @thorstenMueller noted the best option is using original WAV and not using lossy compression (like MP3 is) at all, though.
